2022-03-28 15:13:33,540 INFO    MainThread:17253 [wandb_setup.py:_flush():71] setting env: {}
2022-03-28 15:13:33,540 INFO    MainThread:17253 [wandb_setup.py:_flush():71] setting login settings: {}
2022-03-28 15:13:33,540 INFO    MainThread:17253 [wandb_init.py:_log_setup():371] Logging user logs to /Users/kiranganeshan/reading-group/wandb/run-20220328_151333-2ldaxubv/logs/debug.log
2022-03-28 15:13:33,540 INFO    MainThread:17253 [wandb_init.py:_log_setup():372] Logging internal logs to /Users/kiranganeshan/reading-group/wandb/run-20220328_151333-2ldaxubv/logs/debug-internal.log
2022-03-28 15:13:33,541 INFO    MainThread:17253 [wandb_init.py:init():404] calling init triggers
2022-03-28 15:13:33,541 INFO    MainThread:17253 [wandb_init.py:init():411] wandb.init called with sweep_config: {}
config: {}
2022-03-28 15:13:33,541 INFO    MainThread:17253 [wandb_init.py:init():449] starting backend
2022-03-28 15:13:33,541 INFO    MainThread:17253 [backend.py:_multiprocessing_setup():97] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-03-28 15:13:33,559 INFO    MainThread:17253 [backend.py:ensure_launched():199] starting backend process...
2022-03-28 15:13:33,567 INFO    MainThread:17253 [backend.py:ensure_launched():205] started backend process with pid: 17278
2022-03-28 15:13:33,569 INFO    MainThread:17253 [wandb_init.py:init():458] backend started and connected
2022-03-28 15:13:33,574 INFO    MainThread:17253 [wandb_init.py:init():519] updated telemetry
2022-03-28 15:13:33,590 INFO    MainThread:17253 [wandb_init.py:init():549] communicating current version
2022-03-28 15:13:34,243 INFO    MainThread:17253 [wandb_init.py:init():554] got version response upgrade_message: "wandb version 0.12.11 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-03-28 15:13:34,243 INFO    MainThread:17253 [wandb_init.py:init():564] communicating run to backend with 30 second timeout
2022-03-28 15:13:34,368 INFO    MainThread:17253 [wandb_init.py:init():592] starting run threads in backend
2022-03-28 15:13:37,180 INFO    MainThread:17253 [wandb_run.py:_console_start():1816] atexit reg
2022-03-28 15:13:37,181 INFO    MainThread:17253 [wandb_run.py:_redirect():1690] redirect: SettingsConsole.REDIRECT
2022-03-28 15:13:37,181 INFO    MainThread:17253 [wandb_run.py:_redirect():1695] Redirecting console.
2022-03-28 15:13:37,192 INFO    MainThread:17253 [wandb_run.py:_redirect():1751] Redirects installed.
2022-03-28 15:13:37,193 INFO    MainThread:17253 [wandb_init.py:init():619] run started, returning control to user process
2022-03-28 15:13:37,194 INFO    MainThread:17253 [wandb_run.py:_atexit_cleanup():1786] got exitcode: 1
2022-03-28 15:13:37,195 INFO    MainThread:17253 [wandb_run.py:_restore():1758] restore
2022-03-28 15:13:39,708 INFO    MainThread:17253 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 690
  total_bytes: 690
}

2022-03-28 15:13:40,091 INFO    MainThread:17253 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 690
  total_bytes: 690
}

2022-03-28 15:13:40,416 INFO    MainThread:17253 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 690
  total_bytes: 4412
}

2022-03-28 15:13:40,520 INFO    MainThread:17253 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 690
  total_bytes: 4412
}

2022-03-28 15:13:40,626 INFO    MainThread:17253 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 690
  total_bytes: 4412
}

2022-03-28 15:13:40,732 INFO    MainThread:17253 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 690
  total_bytes: 4412
}

2022-03-28 15:13:40,835 INFO    MainThread:17253 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 690
  total_bytes: 4412
}

2022-03-28 15:13:40,942 INFO    MainThread:17253 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 690
  total_bytes: 4412
}

2022-03-28 15:13:41,050 INFO    MainThread:17253 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 4412
  total_bytes: 4412
}

2022-03-28 15:13:41,154 INFO    MainThread:17253 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 4412
  total_bytes: 4412
}

2022-03-28 15:13:41,259 INFO    MainThread:17253 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 4412
  total_bytes: 4412
}

2022-03-28 15:13:41,365 INFO    MainThread:17253 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 4412
  total_bytes: 4412
}

2022-03-28 15:13:41,621 INFO    MainThread:17253 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 4412
  total_bytes: 4412
}

2022-03-28 15:13:42,081 INFO    MainThread:17253 [wandb_run.py:_wait_for_finish():1916] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 4412
  total_bytes: 4412
}
local_info {
}

2022-03-28 15:13:43,197 INFO    MainThread:17253 [wandb_run.py:_append_files():2184] logging synced files
